vhdbuilder/packer/init-variables.sh

```diff
@@ -211,6 +211,14 @@ if [ -n "${WINDOWS_MSI_RESOURCE_STRING}" ]; then
 	windows_msi_resource_strings+=(${WINDOWS_MSI_RESOURCE_STRING})
 fi
 
+linux_msi_resource_ids=()
+if [ -n "${LINUX_MSI_RESOURCE_ID}" ]; then
+	echo "LINUX_MSI_RESOURCE_ID is set in pipeline variables: ${LINUX_MSI_RESOURCE_ID}"
+	linux_msi_resource_ids+=(${LINUX_MSI_RESOURCE_ID})
+fi
+
+private_packages_url=""
```

Comment:
nit: move declaration to L384 right before we append `PRIVATE_PACKAGES_URL` to it?